src/components/Header/HeaderContact.js

```diff
@@ -90,12 +90,16 @@ const styles = StyleSheet.create({
   }),
   text: {
     color: COLORS.black,
-    fontFamily: fonts.inter[600],
+    fontFamily: fonts.inter[500],
     fontSize: normalizeFontSize(14),
-    fontWeight: 'bold',
     padding: 10,
     textAlign: 'center'
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The font family has been changed from `fonts.inter[600]` to `fonts.inter[500]`, and the `fontWeight` property has been removed. This could potentially alter the appearance of the text. Make sure this change is consistent with the overall design language of your application.
```diff
- fontFamily: fonts.inter[600],
- fontWeight: 'bold',
+ fontFamily: fonts.inter[500],
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->